package.json

```diff
@@ -22,50 +22,68 @@
   },
   "license": "MIT",
   "author": "Flybondi <dev@flybondi.com> (https://flybondi.com)",
+  "sideEffects": false,
```

Comment:
gold plating? is this related with https://github.com/flybondi/ramda-land/issues/16

Reply:
It is. Harmless for now.